resources/js/src/app/components/itemList/filter/FindologicItemFilter.spec.ts

```diff
@@ -0,0 +1,351 @@
+import Vuex from 'vuex';
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils';
+import { Facet, ColorFacet, State } from '../../../shared/interfaces';
+import { Store } from 'vuex';
+import FindologicItemFilter from './FindologicItemFilter.vue';
+import VueCompositionAPI from '@vue/composition-api';
+import UrlBuilder from '../../../shared/UrlBuilder';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+localVue.use(VueCompositionAPI);
+
+window.ceresTranslate = key => key;
+
+describe('FindologicItemFilter', () => {
+
+  let store: Store<State>;
+
+  beforeEach(() => {
+    store = new Vuex.Store({});
+  });
+
+  it('Shows the facet name', () => {
+    const facet: Facet = {
+      cssClass: '',
+      findologicFilterType: 'select',
+      id: 'test',
+      isMain: false,
+      itemCount: 3,
+      name: 'Facet name',
+      noAvailableFiltersText: '',
+      select: 'multiple',
+      type: '',
+      values: [
+        {
+          count: 9,
+          id: '20',
+          image: '',
+          items: [],
+          name: '22220',
+          position: 'item',
+          selected: false
+        },
+        {
+          count: 1,
+          id: '21',
+          image: '',
+          items: [],
+          name: '22221',
+          position: 'item',
+          selected: false
+        }
+      ]
+    };
+
+    store.state.itemList = {
+      isLoading: false,
+      selectedFacets: [],
+      facets: [facet]
+    };
+
+    const wrapper = shallowMount(FindologicItemFilter, { propsData: { 'facet': facet, 'filtersPerRow': 3 }, store, localVue });
```

Comment:
With JS you don't need to specify the index key, if the variable name would be name of the key. 
```suggestion
    const wrapper = shallowMount(FindologicItemFilter, { propsData: { facet, 'filtersPerRow': 3 }, store, localVue });
```